noetic-foxy/Dockerfile

```diff
@@ -198,8 +199,6 @@ RUN sed -i 's|http://archive.ubuntu.com|http://us.archive.ubuntu.com|g' /etc/apt
         sh -c 'echo "deb [trusted=yes] https://s3.amazonaws.com/autonomoustuff-repo/ $(lsb_release -sc) main" > /etc/apt/sources.list.d/autonomoustuff-public.list' && \
         apt-get update && \
         apt-get install --no-install-recommends --yes libas-common && \
-        # ROS2 SSC dependencies from Astuff
-        apt-get install ros-foxy-pacmod3-msgs --yes && \
```

Comment:
I think this is failing because the source list from autonomous stuff need to be first?
Also that makes me think why it didn't work the first time because the message dependency is here?

Reply:
Yes that is correct, this package is from Astuff distribution